include/aos/sm/instance.hpp

```diff
@@ -83,17 +83,17 @@ class Instance {
     const Error& RunError() const { return mRunError; };
 
     /**
-     * Returns instance service Aos version.
+     * Returns instance service version.
      *
-     * @return uint64_t Aos version.
+     * @return StaticString<cVersionLen> version.
      */
-    uint64_t AosVersion() const
+    StaticString<cVersionLen> GetServiceVersion() const
     {
         if (mService) {
-            return mService->Data().mVersionInfo.mAosVersion;
+            return mService->Data().mVersion;
         }
 
-        return 0;
+        return "";
```

Comment:
should we return some default semver in this case, like "0.0.0" ?

Reply:
Interesting question. Let's return empty string for now. Then will see.